python-avd/pyavd/_eos_designs/structured_config/network_services/router_ospf.py

```diff
@@ -106,6 +106,13 @@ def _update_ospf_interface(
                         continue
                     process.no_passive_interfaces.append(l3_interface.interfaces[node_index])
 
+        for l3_port_channel in vrf.l3_port_channels:
+            if l3_port_channel.ospf.enabled:
+                for _node_index, node in enumerate(l3_port_channel.nodes):
+                    if node != self.shared_utils.hostname:
+                        continue
```

Comment:
I don't think you need this. Remember the l3_port_channels are already filtered inside `filtered_tenants` so we only get the ones that contain our device somewhere.